boxsdk/config.py

```diff
@@ -9,3 +9,9 @@ class API(object):
     UPLOAD_URL = 'https://upload.box.com/api/2.0'
     OAUTH2_API_URL = 'https://api.box.com/oauth2'  # <https://developers.box.com/docs/#oauth-2>
     OAUTH2_AUTHORIZE_URL = 'https://app.box.com/api/oauth2/authorize'  # <https://developers.box.com/docs/#oauth-2-authorize>
+
+
+class Version(object):
+    """Configuration object containing the package version and user agent string."""
+    VERSION = '1.5.0'
```

Comment:
Now we have duplication between here and setup.py.

Can you do something like this:
- Create boxsdk/version.py, with contents
```python
__version__ = '1.5.0'
```
- In this file, ``from .version import __version__``, then ``VERSION = __version__``.
- In boxsdk/__init__.py, you can do ``from .version import __version__`` instead of ``__version__ = Version.VERSON``.
- In setup.py, ``open('boxsdk/version.py')`` and read the version constant. E.g. https://github.com/kennethreitz/requests/blob/master/setup.py#L50